submissions/webdevtoday/task_js_dom/js/app.js

```diff
@@ -0,0 +1,35 @@
+import {Controller} from './Controller.js';
+import {Route} from './Route.js';
+
+document.addEventListener('DOMContentLoaded', () => {
+    'use strict';
+    
+    const navRoutes = document.querySelector('[data-type="routes"]');
+    const mainContent = document.querySelector('main');
+
+    const controller = new Controller(mainContent);
+    const route = new Route(controller, {
```

Comment:
You can create some sonstant for storing an object with the routes, but your approach is ok as well

Reply:
Ok